pkgs/tools/misc/fortune/extensions/blag-fortune.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, stdenv, fetchgit, fortune }:
+
+stdenv.mkDerivation rec {
+  pname = "blag-fortune";
+  version = "2022-07-16";
+
+  # We use fetchurl instead of fetchFromGitHub because the release pack has some
+  # special files.
+  src = fetchgit {
+    url = "https://notabug.org/PangolinTurtle/BLAG-fortune.git";
+    rev = "b07fb0312f";
+    sha256 = "sha256-Fi8o+hd2frBXXfzjwiJC7/2ZjdEc8vJbB7bSh9n3Vx0=";
+  };
+
+  # NOTE: this is needed for strfile in installPhase
+  buildInputs = [ fortune ];
+
+  installPhase = ''
```

Comment:
If you redefine phases, you should also add `runHook`s to it, for instance for the `installPhase` it should be:
```
runHook preInstall

# Do stuff.

runHook postInstall
```